src/content/developers/docs/storage/index.md

```diff
@@ -74,7 +71,7 @@ Types of dStorage with a challenge mechanism:
 
 ### Decentrality {#decentrality}
 
-There aren't great tools to measure the level of decentralization of platforms, but in general you'll want to use tools that don't have some form of KYC to prove they are actually decentralized.
+There aren't great tools to measure the level of decentralization of platforms, but in general, you'll want to use tools that don't have some form of KYC to prove they are genuinely decentralized.
```

Comment:
Hesitate on this one, since lack of KYC certainly doesn't really prove anything. My thought would be to reword that lack of KYC can provide evidence that a project is not centralized, while staying away from "prove."

```suggestion
There aren't great tools to measure the level of decentralization of platforms, but in general, you'll want to use tools that don't have some form of KYC to provide evidence they are not centralized.
```

Reply:
Yeah on second read you're 100% right this is a bit misleading.